### PR TITLE
add typing for wepy

### PR DESCRIPTION
**Pull request by jimexist**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).
- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [x] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] `tslint.json` should be present, and `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

**Review by plantain-00 — CHANGES_REQUESTED** [review]
it should export default: https://github.com/Tencent/wepy/blob/1.7.x/packages/wepy/src/wepy.js#L19

**Comment by Jimexist** [comment]
@plantain-00 default export added

File: types/wepy/wepy-tests.ts
Comment (plantain-00) [inline types/wepy/wepy-tests.ts]: It should be `import wepy from 'wepy'` https://github.com/Tencent/wepy#demo

File: types/wepy/index.d.ts
Comment (plantain-00) [inline types/wepy/index.d.ts]: I think you can just copy the code from its source code

```ts
export default {
    event: event,
    app: app,
    component: component,
    page: page,
    mixin: mixin,

    $createApp: base.$createApp,
    $createPage: base.$createPage,

    $isEmpty: util.$isEmpty,
    $isEqual: util.$isEqual,
    $isDeepEqual: util.$isDeepEqual,
    $has: util.$has,
    $extend: util.$extend,
    $isPlainObject: util.$isPlainObject,
    $copy: util.$copy,
};
```

**Review by plantain-00 — CHANGES_REQUESTED** [review]

**Comment by Jimexist** [comment]
@plantain-00 thanks for the comment, now it's updated to reflect default export

**Review by plantain-00 — APPROVED** [review]

**Comment by Jimexist** [comment]
@plantain-00 @typescript-bot who else should i ping in order to get approvals and merge?

**Comment by plantain-00** [comment]
https://github.com/DefinitelyTyped/DefinitelyTyped#ive-submitted-a-pull-request-how-long-until-it-is-merged

**Pull request merged**
